### PR TITLE
feat: add codespell lint job into workflows & fix typos

### LINKED ISSUE #108: feat: add codespell lint job into workflows
**Is your feature request related to a problem? Please describe.**

Codespell linter tool is very important for code and system quality.

**Describe the solution you'd like**

So I want to add codespell lint job into workflows.

### PR DESCRIPTION
**Pull request by vuittont60**
Closes #108 

1. feat: add codespell lint job into workflows
2. fix typos caught by lint job.

Please help me to review. Thank you very much.

### CODE REVIEW

**Comment by 22388o** [comment]
Thanks for PR @vuittont60 !

We're analyzing :)

File: src/web/command.sh
Comment (22388o) [inline src/web/command.sh:1]: I don't understand this. You moved for other file?

Comment (vuittont60) [reply src/web/command.sh:1]: rename src/web/comand.sh to src/web/command.sh

Comment (vuittont60) [reply src/web/command.sh:1]: I searched globally and found this file was not invoked by others, so I just renamed it.

Comment (22388o) [reply src/web/command.sh:1]: Ok

**Pull request merged**